shaderlib/fp64/cos-fp64.glsl

```diff
@@ -91,6 +91,21 @@ vec2 cos_fp64(vec2 a) {
     vec2 u = vec2(0.0, 0.0);
     vec2 v = vec2(0.0, 0.0);
 
+#ifdef NVIDIA_WORKAROUND
```

Comment:
I think we should either just call this flag `NVIDIA` - or be more specific, `NVIDIA_COMPARISON_WORKAROUND`